### DISCUSSION TITLE
Connect API Documentation

### DISCUSSION THREAD

**Discussion by chrisopoeia** [matin/garth#56] [General]
Hi, firstly thanks so much for providing garth, I am using it via garminconnect on a project.

Just wondered, do you know where I can find any documentation for the Connect API please?

I am trying to get activity data with elevation corrections enabled and wonder if this is possible via the API, but I can't find any docs for the endpoints, have exhausted all search ideas I can think of :(

**Comment by matin** [comment]
Hi @chris-brown-nz, there are no official documentation.

I recommend running the mobile app through Charles while opening the app screen that shows the information you want. Charles will show you the request that's being made.

If you're using iOS, the easiest way to run Charles is with the iOS app. If you're using Android, you can run Charles on your computer and proxy your mobile requests through your computer.
